template/README__labs.md

```diff
@@ -38,9 +38,8 @@ import "[npm]/[namespace]-[name]";
 
 **Reference component in HTML**
 
-```html
-<[namespace]-[name]>Hello World</[namespace]-[name]>
-```
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./apiExamples/basic.md) -->
```

Comment:
```suggestion
<!-- AURO-GENERATED-CONTENT:START (CODE:src=../demo/apiExamples/basic.html) -->
```

Reply:
See below comment for why I don't want to move the apiExamples

Reply:
See below comment for why I don't want to move the apiExamples